### DISCUSSION TITLE
Centos7 error

### DISCUSSION THREAD

**Discussion by jesse-p** [madMAx43v3r/chia-plotter#112] [Q&A] [answered]
Hello, I used centos7 to compile and install when I reported an error:

`
/chia-plotter/lib/bls-signatures/src/util.hpp:101:15: error: ‘invalid_argument’ is not a member of ‘std’
         throw std::invalid_argument("Invalid input string");
...
/chia-plotter/build/_deps/relic-src/src/rand/relic_rand_hashd.c: In function ‘rand_hash’:
/opt/chia-plotter/build/_deps/relic-src/src/rand/relic_rand_hashd.c:71:2: error: ‘for’ loop initial declarations are only allowed in C99 mode
  for (int i = 0; i < len; i++) {
`
Is centos7 supported?
Can help look at this problem? thx!

**Comment by madMAx43v3r** [comment]
Centos7 is way too old sry

**Comment by madMAx43v3r** [comment, marked as answer]
Well it seems somebody managed to do it here: https://github.com/madMAx43v3r/chia-plotter/pull/118/files

**Reply by jesse-p** [reply]
thx very much! It's OK.

**Comment by Hardcore-fs** [comment]
There is nothing wrong with centos 7
I guess you have NOT followed the build instructions for centos7
the instructions are the ones I released to get it to build.

https://github.com/madMAx43v3r/chia-plotter/issues/96

**Reply by jesse-p** [reply]
Yes, you're right.
